public/datasets/thanksgiving.json

```diff
@@ -0,0 +1,205 @@
+{
+  "card": {
+    "description": "A survey from FiveThrityEight, asking people around the country what foods they have for Thanksgiving dinner.",
```

Comment:
maybe "asking people across the United States"? It's a US-centric dataset and I'm being nit-picky but know we want to think in terms of a global audience as we continue to expand worldwide usage?